greenboot.spec

```diff
@@ -172,7 +194,20 @@ install -DpZm 0755 etc/greenboot/check/wanted.d/* %{buildroot}%{_sysconfdir}/%{n
 %{_libexecdir}/%{name}/redboot-auto-reboot
 %{_unitdir}/redboot-auto-reboot.service
 
+%files update_platforms_check
+%{_libexecdir}/%{name}/greenboot-loading-message
+%{_sysconfdir}/%{name}/check/required.d/01_repository_dns_check.sh
+%{_sysconfdir}/%{name}/check/wanted.d/01_update_platforms_check.sh
+%{_sysconfdir}/systemd/system/greenboot-healthcheck.service.d/10-network-online.conf
+%{_unitdir}/greenboot-loading-message.service
```

Comment:
I wonder whether the loading-message service should just be part of the main package, as it might be useful in other contexts than just this specific case.

Reply:
I think that's a fair point, moving it it to the main package!